images/eslint/package.json

```
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "4.8.2",
+    "@typescript-eslint/eslint-plugin": "4.9.0",
     "@typescript-eslint/parser": "4.8.2",
```

Could you keep the same version, please?

```suggestion
    "@typescript-eslint/parser": "4.9.0",
```

---

I've added a new Goodcheck rule: #1714

---

👍 That could be useful.

---

The latest version 4.9.1 of ` @typescript-eslint/parser` has just been issued at #1713. I will update the version of **eslint-plugin** in that PR accordingly, and mark this PR as DO NOT MERGE.

---

`@dependabot recreate` can help us in such a case.